lld/test/ELF/avr-errors.s

```diff
@@ -0,0 +1,27 @@
+; REQUIRES: avr
+; RUN: llvm-mc -filetype=obj -triple=avr -mcpu=atmega328 %s -o %t.o
+; RUN: not ld.lld %t.o -o %t 2>&1 -Ttext=0x1000 --defsym=callee0=0xf00 --defsym=callee1=0x2048 --defsym=callee2=0x100f | FileCheck %s
```

Comment:
Use `--implicit-check-not=error:` to catch undesired errors, rather than use a lot of `-NOT:`

Reply:
Thanks! I have fixed.